test_unstructured_ingest/dest/qdrant.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -ex
+
+DEST_PATH=$(dirname "$(realpath "$0")")
+SCRIPT_DIR=$(dirname "$DEST_PATH")
+cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=qdrant-dest
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+WORK_DIR=$SCRIPT_DIR/workdir/$OUTPUT_FOLDER_NAME
+max_processes=${MAX_PROCESSES:=$(python3 -c "import os; print(os.cpu_count())")}
+writer_processes=$(((max_processes - 1) > 1 ? (max_processes - 1) : 2))
+CONTAINTER_NAME="qdrant_test"
+QDRANT_PORT=6333
+QDRANT_HOST=localhost:$QDRANT_PORT
+COLLECTION_NAME="qdrant-test-$(date +%s)"
+EXPECTED_POINTS_COUNT=1404
+
+function stop_docker()
+{
+  docker stop $CONTAINTER_NAME
+}
+
+docker run -d --rm \
+           -p 6333:$QDRANT_PORT \
+           --name $CONTAINTER_NAME qdrant/qdrant:latest
+
+trap stop_docker SIGINT
+trap stop_docker ERR
+
+until curl --output /dev/null --silent --get --fail http://$QDRANT_HOST/collections; do
+  printf 'Waiting for Qdrant server to start...'
+  sleep 5
+done
```

Comment:
shouldn't be an issue, but maybe timeout and error after a certain number of retries?

Reply:
5ee1b7f49ea9336b72a313071d5e76d189b8b05d